addons/rivet/devtools/dock/elements/links_container.tscn

```diff
@@ -7,11 +7,14 @@ alignment = 1
 [node name="HubLink" type="LinkButton" parent="."]
 layout_mode = 2
 text = "Hub"
+uri = "https://hub.rivet.gg/"
```

Comment:
This should probably be overridden by the domain that is used in the API field. For example, if I sign in to `api.staging2.gameinc.io`, my hub would be `hub.staging2.gameinc.io`.

Reply:
Just checked with Nathan, we can get the hub link from the bootstrap data. Will check this out once open beta is taken care of 🙌 in the meantime I'd rather merge this and create an issue to look into this later (https://linear.app/rivet-gg/issue/GDT-96/hub-button-should-go-to-linked-hub-in-bootstrap)